packages/ketcher-react/src/script/ui/state/server/index.js

```diff
@@ -157,7 +157,7 @@ export function serverTransform(method, data, struct) {
         )
       })
       .catch(e => {
-        state.editor.errorHandler(e.message)
+        state.editor.errorHandler(e.message || e)
```

Comment:
seems, errorHandler expects only strings (not objects)

Reply:
It seems like sometimes e can be string